Abstraction/3/Author.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Abstraction
+{
+    internal class Author : Book
```

Comment:
Book is not an abstraction for this Class!
It contains all these parts... But abstraction should be another class

Reply:
And all classes have the same property it should be in abstraction